app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -28,7 +28,7 @@
     <% end %>
     <% if article.published %>
       <span id="pageviews-<%= article.id %>" class="cta pill dashboard-pageviews-indicator" data-analytics-pageviews data-article-id="<%= article.id %>">
-        <%= article.page_views_count > 100 ? article.page_views_count : "under 100" %> views // <%= article.positive_reactions_count %> reactions // <%= article.comments_count %> comments
+        <%= article.page_views_count > 100 ? article.page_views_count : "under 100" %> views // <%= article.positive_reactions_count %> reaction<%= article.positive_reactions_count != 1 ? "s" : "" %> // <%= article.comments_count %> comment<%= article.comments_count != 1 ? "s" : "" %>
```

Comment:
Hi @selbekk, good catch, you could consider using [pluralize](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize) instead of manual counts

Reply:
I could, but I have no ruby experience at all, so I’m not sure how i’d do that

Reply:
@rhymes is text_helper already imported?

Reply:
`pluralize` takes an count as and the singular string like this `pluralize(2, 'person')` and then will return the correct (usually) pluralization of the word `people`.

In this case the count is a dynamic value so it would look like this

`pluralize(article.positive_reactions_count, 'reaction')`

I then needs to be interpolated in the HTML template so you can wrap it in `<%= %>`

`<%= pluralize(article.positive_reactions_count, 'reaction') %>`

Reply:
Interestingly, there's not really much in it, apart from style of course.

```ruby
<%= pluralize(article.positive_reactions_count, "reaction") %>
reaction<%= article.positive_reactions_count != 1 ? "s" : "" %>
reaction<%= article.positive_reactions_count != 1 && "s" %>
```

Reply:
@selbekk no worries, I can explain: basically `pluralize` is a Rails helper, a method injected in the template view to help formatting. It works with two parameters, the first is the count and the second is the string that has the possible plural, in our case:

* `article.positive_reactions_count` for "reaction"
* `article.comments_count` for the word "comment"

To use a helper you call it inside the "outputting" tags, for example, instead of 

```erb
reaction<%= article.positive_reactions_count != 1 ? "s" : "" %>
```

you could call

```erb
<%= pluralize(article.positive_reactions_count, "reaction") %>
```
This way the framework will decide to use the singular or plural form based from the value of count. The main difference are readability, the fact that you can use irregular forms or different words by explicitly passing the plural:

```erb
<%= pluralize(article.positive_reactions_count, "reaction", plural: "reactions") %>
```
and support for multiple locale (not a concern in this case I think)

Hope this helps :)

Reply:
Rails overloads `String` with pluralize so you can also do `'reaction'.pluralize(count: 2)`.

Reply:
```ruby
<%= "reaction".pluralize(count: article.positive_reactions_count) %>
```